app/src/main/java/com/babylon/wallet/android/presentation/account/createaccount/CreateAccountNav.kt

```diff
@@ -87,9 +87,9 @@ fun NavGraphBuilder.createAccountScreen(
                 type = NavType.StringType
                 nullable = true
             },
-            navArgument(ARG_NETWORK_NAME) {
-                type = NavType.StringType
-                nullable = true
+            navArgument(ARG_NETWORK_ID) {
+                type = NavType.IntType
+                defaultValue = -1 // no network id, use current network
```

Comment:
Could we not make this Int nullable like the String above instead of using -1 ? 🤔

Reply:
Ints can't be nullable in nav arguments, wanted to do this as well

Reply:
@raf-rdx used constant instead